FragmentService/FragmentServiceInterface.php

```diff
@@ -46,6 +46,15 @@ public function buildCreateForm(FormMapper $form, FragmentInterface $fragment);
     public function buildForm(FormMapper $form, FragmentInterface $fragment);
 
     /**
+     * Gets content to render fragment.
```

Comment:
What is a content?

Reply:
this method is king of similar to the `execute` method of BlockService for fragment except that it returns an array for the fragment template. By default, `fragment` and  `fields` will be transmitted but it can be completed in each fragment service defined by final user